### PR TITLE
updating README.md for v3.0.0

### PR DESCRIPTION
**Pull request by paulypeter**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,12 +96,12 @@ port=465
 empfaenger=beispielsender@server.tld,beispielsmfaenger@server.tld
 
 [TELEGRAM]
-enable_telegram=true
+enable_=true
```
Comment (jnslmk) [inline README.md:99]:
```suggestion
enable=true
```

**Review by jnslmk — CHANGES_REQUESTED** [review]
Typo

File: README.en.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+# Notification bot for the Lower Saxony vaccination portal 🐴
+
+![GitHub tag (latest by date)](https://img.shields.io/github/v/tag/sibalzer/impfbot?label=version)
+[![Python](https://img.shields.io/badge/Made%20with-Python%203.x-blue.svg?style=flat-square&logo=Python&logoColor=white)](https://www.python.org/)
+[![GitHub license](https://img.shields.io/github/license/sibalzer/impfbot)](https://github.com/sibalzer/impfbot/blob/main/LICENSE)
+
+A little weekend project of mine. The bot monitors the REST-API of the lower saxony vaccination portal (https://impfportal-niedersachsen.de) for free vaccination slots and sends a notification via mail. From then on, unfortunately, the fastest wins. Please do not abuse the bot and use moderate intervals.
+
```
Comment (paulypeter) [inline README.en.md:8]: @sibalzer Do you want a link to the German (default) version here?

Comment (sibalzer) [reply README.en.md:8]: Would be great

Comment (paulypeter) [reply README.en.md:10]: Done

**Comment by sibalzer** [comment]
config.example.ini needs to be updated too

**Comment by paulypeter** [comment]
The `enable` flags are already correct there, I now added the group size setting.

**Review by sibalzer — APPROVED** [review]

**Pull request merged**